website/docs/r/teams.html.markdown

```diff
@@ -1,9 +1,5 @@
 ---
-layout: "mongodbatlas"
-page_title: "MongoDB Atlas: teams"
-sidebar_current: "docs-mongodbatlas-resource-teams"
-description: |-
-    Provides a Team resource.
+subcategory: "Deprecated"    
 ---
```

Comment:
question: do we need a WARNING note here to instruct users which resource they should use instead? we seem to have similar notes for the other deprecated resources above.